spec/src/main/asciidoc/converters.asciidoc

```diff
@@ -76,18 +76,18 @@ If an Implicit Converter cannot convert a value, a `java.lang.IllegalArgumentExc
 For the built-in converters and custom converters, the corresponding Array converters are provided by default.
 The delimiter for the config value is ",".
 The escape character is "\".
-e.g. With this config `myPets=dog,cat,dog\,cat`, the values as an array will be `{"dog", "cat", "dog,cat"}`.
+e.g. With this config `myPets=dog,cat,dog\,cat`, the values as an array will be `{"dog", "cat", "dog", "cat"}`.
```

Comment:
Not quite, as the third comma was skipped. dog,cat will be a single value.

Reply:
I understood this part to be only a value.

however, the output does not match what is passed in SPEC.

  I may have misunderstanding, but should not it be [dog, "cat," "dog, cat"] with 3 positions?

  because in the image above, it shows 4 positions.

thanks